src/Pages/Layout/Layout.js

```diff
@@ -0,0 +1,98 @@
+import React, { PureComponent } from 'react';
+import ReactGA from 'react-ga';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import AsyncComponent from '../Utils/AsyncComponent';
+import Header from './Header/Header';
+import Isna from './Isna/Isna';
+import * as styles from './Layout.module.css';
+
+class Layout extends PureComponent {
+  render() {
+    return (
+      <BrowserRouter>
+        <div
+          className={`${styles.bgGrayAlternative} ${styles.paddingTop68} ${
+            styles.stretch
+          } black-alternative lh-title sans-serif`}
+        >
+          <Header />
+          <Route
+            component={() => {
```

Comment:
This should be extracted from render, as it's not using anything from the closure, doing this will also improve the readibility